onnxruntime/core/graph/ep_api_types.cc

```diff
@@ -248,6 +248,36 @@ Status EpNode::GetAttributes(gsl::span<const OrtOpAttr*> dst) const {
   return Status::OK();
 }
 
+Status EpNode::GetTensorAttributeAsOrtValue(const OrtOpAttr* attribute, OrtValue** result) const {
+  const auto* attr_proto = reinterpret_cast<const ONNX_NAMESPACE::AttributeProto*>(attribute);
+
+  if (attr_proto->type() != onnx::AttributeProto::TENSOR) {
+    *result = nullptr;
+    return ORT_MAKE_STATUS(ONNXRUNTIME, FAIL, "This OrtOpAttr instance is not a 'TENSOR' attribute");
+  }
+
+  auto tensor_proto_to_ort_value = [&](const ONNX_NAMESPACE::TensorProto& tensor_proto,
+                                       OrtValue** result) -> Status {
+    const auto& graph_viewer = ep_graph_->GetGraphViewer();
+
+    // Initialize OrtValue for tensor attribute.
+    // Note: using std::unique_ptr<OrtValue> because we return a OrtValue* to the user and we want it to be stable.
+    auto tensor_attribute_value = std::make_unique<OrtValue>();
```

Comment:
nit: this comment about unique_ptr seems unnecessary now.

also, I'd slightly prefer keeping the unique_ptr at the function scope instead of in this lambda. we can call unique_ptr::release() at the end of the function and it is obvious that we won't leak before that. perhaps we don't need the lambda and we can just put its logic inline.